contracts/strategies/fluid/interfaces/IfToken.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+interface IfToken {
```

Comment:
`IfToken` looks weird, can we rename it to `IFluidToken` instead.

Reply:
I had a similar thought, sounds like If token. Good suggestion though, updated it here: 4c06c5cbe001179619e4089ba709e67b67dd7a41